src/core/theme/reset/index.ts

```diff
@@ -27,18 +27,16 @@ export const font = (props: SuomifiThemeProp) => (
   ${props.theme.typography[typographyToken]}
 `;
 
-export const input = (props: SuomifiThemeProp) => {
-  return css`
-    ${element(props)}
-    ${font(props)('actionElementInnerText')}
+export const input = (props: SuomifiThemeProp) => css`
+  ${element(props)}
+  ${font(props)('actionElementInnerText')}
     min-width: 245px;
```

Comment:
Maybe some problem with prettier here as the indent is off for `min-width`?

Reply:
Yes, this is an existing issue with styles that don't end up with semicolon. It's possible that prettier / stylellint reintroduced some issues that were already fixed. At least previously it was possible to manually fix some of these without automatic reformatting on save.

Reply:
Could we add the missing semicolons then, `;`?

At least then it seems to indent it correctly.

Reply:
Fixed manually.

Reply:
Adding missing semicolons would lead to double semicolons for the last returned rule in the actual output. Since these abstractions return multiple css rules we cannot omit the semicolons in the returns. Or we could omit the last semicolon, but that would be risky in my opinion.